### ISSUE TITLE
Syslog connector is missing from the catalog

### ISSUE THREAD

**Issue by apupier** [apache/camel-kafka-connector#677]
`org.apache.camel.kafkaconnector.syslog.CamelSyslogSourceConnector` is not part of the catalog.

guess: the class extends CamelNettySourceConnector and not directly CamelSourceConnector or it is because the class is not generated

**Comment by oscerd** [comment]
The syslog connector is a bit particular. So, no, it's not part of autogenerated catalog.

**Comment by apupier** [comment]
is it the only one?

any ideas to include "particular" connector in catalog?

**Comment by oscerd** [comment]
Yes, it is the only one. Particular is related to the fact that basically the component is a dataformat. To make it work as a connector we needed to make it extending the NettySourceConnector. This was done in the early stage of the development while we were testing what could work and what not.

**Comment by oscerd** [comment]
The only way we have to do this is including it in the catalog by hand.
